docs/source/app_developers_guide/intro_xo_transaction_family.rst

```diff
@@ -0,0 +1,127 @@
+
+1. After starting up a validator, *XO* transaction processor, and the REST API (see `Installing and Running Sawtooth <http://intelledger.github.io/0.8/cli/xo.html>`_), 
+   each player must initialize *XO* locally using the ``xo init`` command. Initialization
+   information is stored locally in a file. This command has two options: ``--username``
+   and ``--url``.
+
+  - The ``--username`` option is the name of the player. If ``--username`` is not
+    specified, OS environment variables will be parsed to find a suitable value.
+  - The ``--url`` option should be set to the URL of the REST API. The *XO* client 
+    will send requests to this URL in order to determine the validity of each move and 
+    update the game state on the blockchain. If no URL is specified, the URL will 
+    default to ``http://127.0.0.1:8080``.
+  
+    .. note::
+
+      A user can use the ``xo reset`` command to delete local *XO* data. This includes 
+      the saved URL and username.
+
+2. A user creates a new game using the ``xo create [name]`` command.
+  
+  - The *name* argument is the identifier for the new game to be created.
+
+3. Players alternate turns using the ``xo take [name] [space]`` command to mark spaces on
+   the grid.
+
+  - The *name* argument is the identifier for an existing game.
+  - The *space* argument is the space that is to be marked, which is an integer in the
+    range of 1 through 9.
+
+    .. note::
+
+      The first player to issue an ``xo take`` command to a newly created game is
+      recorded by their username as *player one*. The second player to issue a ``take`` 
+      command is recorded by their username as *player two*.
+
+      Each time a user attempts to take a space, the transaction processor will verify
+      that their username matches the name of the player whose turn it is. This ensures 
+      that no player is able to mark a space out of turn.
+
+4. The *XO* transaction processor will scan for winning or draw board conditions after
+   each turn. If either condition occurs, further ``take`` actions on the finished game 
+   will not be allowed.
+
+.. note::
+
+  If you do not have access to a gaming partner, it is possible to play *XO* against 
+  yourself! To play *XO* on a single machine, simply enter the ``xo init`` command with 
+  default settings. Since *XO* assigns players on a "first come, first served" basis, 
+  your username will be associated with both *player one* and *player two*, allowing you 
+  to issue commands for both players with no additional configuration.
+
+Viewing the game state
+----------------------
+
+The *XO* CLI also supports commands to view the state of ongoing or finished games.
+These commands are ``xo show [game]`` and ``xo list``
+
+- ``xo show [game]`` shows the board of a specified game, as well as data about the 
+  players and game state.
+- ``xo list`` shows a list of all ongoing and finished games that are stored in the 
```

Comment:
drop "that are stored in the state dictionary." because it is an implementation detail